runatlantis.io/docs/using-atlantis.md

```diff
@@ -60,7 +81,6 @@ If you always need to append a certain flag, see [Custom Workflow Use Cases](cus
 
 ---
 ## atlantis apply
-![Apply Command](./images/pr-comment-apply.png)
 ```bash
 atlantis apply [options] -- [terraform apply flags]
```

Comment:
We should add that `--disable-auto-merge` flag too. I believe that's an undocumented flag of `atlantis apply` and it can be seen by running `atlantis apply -h`

Reply:
`--auto-merge-disabled` is already described in the Options section.

<img width="572" alt="image" src="https://user-images.githubusercontent.com/915731/208269069-4f69f81d-9fe3-41f3-a087-da130ade188f.png">